Scripts/RemediationScripts/Remediate-EnableSSLDBForMySQLFlexibleServer.ps1

```diff
@@ -0,0 +1,804 @@
+        Write-Host "'Force' flag is provided. SSL will be rolled back on Azure Database for MySQL flexible server(s) in the Subscription without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::SingleDashLine)
+    }
+
+    # List for storing rolled back Service Fabric resource.
+    $DBForMySQLFSRolledBack = @()
+
+    # List for storing skipped rolled back DBForMySQLFS resource.
+    $DBForMySQLFSSkipped = @()
+
+    $validDBForMySQLFSDetails | ForEach-Object {
+        $DBForMySQLFS = $_
+        try {   
+            $RequireSecureTransportRolledBack = Update-AzMySqlFlexibleServerConfiguration -Name $([Constants]::ParameterName)  -ResourceGroupName $_.ResourceGroupName  -ServerName $_.ResourceName -Value $_.RequireSecureTransport
+            $DBForMySQLFSRolledBack += $DBForMySQLFS
+        }
+        catch {
+                $DBForMySQLFSSkipped += $DBForMySQLFS
+        }
+    }
+    Write-Host $([Constants]::DoubleDashLine)
+    Write-Host "Rollback Summary:`n" -ForegroundColor $([Constants]::MessageType.Info)
+
+    if ($($DBForMySQLFSRolledBack | Measure-Object).Count -gt 0) {
+        Write-Host "SSL server parameter (require_secure_transport) has been rolled back on the following Azure Database for MySQL flexible server(s) in the Subscription.:" -ForegroundColor $([Constants]::MessageType.Update)
+        $DBForMySQLFSRolledBack | Format-Table -Property $colsProperty -Wrap
+        Write-Host $([Constants]::SingleDashLine)
+
+        # Write this to a file.
+        $DBForMySQLFSRolledBackFile = "$($backupFolderPath)\RolledBackDBForMySQLFS.csv"
+        $DBForMySQLFSRolledBack | Export-CSV -Path $DBForMySQLFSRolledBackFile -NoTypeInformation
+        Write-Host "This information has been saved to" -NoNewline
+        Write-Host " [$($DBForMySQLFSRolledBackFile)]" -ForegroundColor $([Constants]::MessageType.Update) 
+    }
+
+    if ($($DBForMySQLFSSkipped | Measure-Object).Count -gt 0) {
+        Write-Host "Error while rolling back SSL server parameter (require_secure_transport) on Azure Database for MySQL flexible server(s) in the Subscription.:" -ForegroundColor $([Constants]::MessageType.Error)
+        $DBForMySQLFSSkipped | Format-Table -Property $colsProperty -Wrap
+        Write-Host $([Constants]::SingleDashLine)
+
+        # Write this to a file.
+        $DBForMySQLFSSkippedFile = "$($backupFolderPath)\RollbackSkippedDBForMySQLFS.csv"
+        $DBForMySQLFSSkipped | Export-CSV -Path $DBForMySQLFSSkippedFile -NoTypeInformation
+        Write-Host "This information has been saved to" -NoNewline
+        Write-Host " [$($DBForMySQLFSSkippedFile)]" -ForegroundColor $([Constants]::MessageType.Update) 
+    }
+}
+function  CheckIfSSLConfigured {
+    param (
+        [String] $RequireSecureTransport 
+    )
+    #Check if server parameter 'require_Secure_transport' is set as ON    
+    if ($RequireSecureTransport -eq $([Constants]::EnableSSLParameterValue)) {
```

Comment:
Are we sure that RequireSecureTransport will always be in Capital letters? like "ON" and "OFF"?